### PR TITLE
Use dependencies.io for dependencies management

### PR DESCRIPTION
**Pull request by Hypnosphi**
Please be prepared to a bunch of PRs after merging this (see https://github.com/Hypnosphi/storybook/pulls). They will be assigned to me though =D

### CODE REVIEW

File: dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+collectors:
+
+# Temporarily disabled
```
Comment (ndelangen) [inline dependencies.yml]: Could this be enabled?

Comment (ndelangen) [reply dependencies.yml]: If not, can you say why it's disabled and what needs to happen to enable it?

Comment (Hypnosphi) [reply dependencies.yml]: I believe the tarballs cause the problem here

Comment (Hypnosphi) [reply dependencies.yml]: See https://github.com/Hypnosphi/storybook/pull/5
Enabling it requires some changes in dependencies.io components

**Comment by ndelangen** [comment]
Please wait for #1908 to be merged

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
OK, here goes nothing..

**Comment by Hypnosphi** [comment]
OK, I hope our CI will survive this

**Pull request merged**